storage/xtradb/srv/srv0mon.cc

```diff
@@ -1970,6 +1973,25 @@ srv_mon_process_existing_counter(
 		value = (mon_type_t) log_sys->lsn;
 		break;
 
+	case MONITOR_PENDING_LOG_WRITE:
+		mutex_enter(&log_sys->mutex);
+		value = static_cast<mon_type_t>(log_sys->n_pending_writes);
+		mutex_exit(&log_sys->mutex);
+		break;
+
+	case MONITOR_PENDING_CHECKPOINT_WRITE:
+		mutex_enter(&log_sys->mutex);
+		value = static_cast<mon_type_t>(
+		    log_sys->n_pending_checkpoint_writes);
+		mutex_exit(&log_sys->mutex);
+		break;
+
+	case MONITOR_LOG_IO:
+		mutex_enter(&log_sys->mutex);
+		value = static_cast<mon_type_t>(log_sys->n_log_ios);
```

Comment:
I would prefer the constructor-style cast:
```c++
 		value = mon_type_t(log_sys->n_log_ios);
```

Reply:
Sorry, I was under the wrong impression that the constructor-style cast is equivalent to `static_cast`. It is actually equivalent to the C-style cast, which has been considered bad style since the C++98 days.